velox/connectors/Connector.h

```diff
@@ -118,6 +118,9 @@ enum class CommitStrategy {
 /// Return a string encoding of the given commit strategy.
 std::string commitStrategyToString(CommitStrategy commitStrategy);
 
+/// Return a commit strategy of the given string encoding.
+CommitStrategy stringToCommitStrategy(const std::string& strategy);
```

Comment:
Sure, done. 

`CommitStrategy` is an `enum class`, in which we can not add static method directly, and `CommitStrategy::kNoCommit/kTaskCommit` is used in `prestissimo` and multiple places in `Velox`, I refactor it as the following `class CommitStrategy ` to be compatible and able to add static method.

```C++
class CommitStrategy {
 public:
  enum CommitStrategyType {
    kNoCommit, // No more commit actions are needed.
    kTaskCommit // Task level commit is needed.
  };

  CommitStrategy(CommitStrategyType commitStrategyType)
      : commitStrategyType_(commitStrategyType) {}

  CommitStrategyType commitStrategyType() const {
    return commitStrategyType_;
  }

  /// Return a string encoding of the given commit strategy.
  static std::string toString(CommitStrategy commitStrategy);

  /// Return a commit strategy of the given string encoding.
  static CommitStrategy parse(const std::string& strategy);

 private:
  CommitStrategyType commitStrategyType_;
};

```

Reply:
It's a little bit overkill.  We can keep `commitStrategyToString` and `stringToCommitStrategy` as top level functions instead of methods of connector in this case.

Reply:
OK, fixed.